packages/image_picker/image_picker/lib/image_picker.dart

```diff
@@ -216,6 +216,7 @@ class ImagePicker {
     int? imageQuality,
     CameraDevice preferredCameraDevice = CameraDevice.rear,
     bool requestFullMetadata = true,
+    bool useImagePickerAndroid = false,
```

Comment:
I think the option being called usePhotoPicker is more accurate since that's the name of the new feature.